package.json

```diff
@@ -34,46 +34,43 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
+    "broccoli-asset-rev": "^2.4.5",
     "chai-jquery": "^2.0.0",
-    "ember-blog-engine": "git://github.com/srowhani/ember-blog-engine.git#caf6e66089b9639cbb076b5eeaa656161f91a5b9",
```

Comment:
@srowhani Is there something specific about the way that `ember-frost-navigation` needs to mount an engine that it needs to have an example of doing so?  Otherwise we're just replicating information about how to use `ember-engines` right, in what appears to be a random location for doing so?  Any other repo could just as easily be used to illustrate how to mount engines, correct?  I'm trying to ascertain if there is something special/specific about the relationship between `ember-frost-navigation` and `ember-engines` that I am not seeing.